lib/purserl.ex

```diff
@@ -35,7 +35,7 @@ defmodule DevHelpers.Purserl do
     state = %{
       port: nil,
       caller: [],
-      purs_cmd: nil,
+      purs_cmd: "purs compile --codegen erl \"lib/**/*.purs\"",
```

Comment:
This won't work for people who have dependencies with Spago.

This fix will need to respect opting in and out of using Spago to be useful.